vizro-core/src/vizro/models/_base.py

```diff
@@ -214,7 +214,7 @@ class VizroBaseModel(BaseModel):
         str,
         AfterValidator(set_id),
         Field(
-            default="",
+            default="",  # TODO[MS]: ask why we decided on empty string vs None
```

Comment:
Probably just to match the annotation `str` I think, otherwise it was messy because you needed `Optional`. Now that pydantic v2 handles `Optional` differently maybe it's better to change though?